templates/home.html

```diff
@@ -1,17 +1,18 @@
 {{ define "content" }}
 
-<h1 class="text-primary">HELLO GOLANG</h1>
-{{ include "partials/flashes" }}
 
-{{ if .data.authenticatedUser }}
-  <h2 class="greeting">Welcome {{ .data.email }}</h2>
-  <h2><a href="/keyword">Keyword</a></h2>
-  <form class="mt-8 space-y-6" action="/logout" method="POST">
-    <button type="submit">Logout</button>
-  </form>
-{{ else }}
-  <h2><a href="/login">Login</a></h2>
-  <h2><a href="/register">Register</a></h2>
-{{ end }}
+  <div class="home">
+    {{ include "partials/flashes" }}
+    
+    <p class="home__title sm:text-5xl md:text-6xl">
```

Comment:
How about using a heading instead e.g. h1 -> h6

Reply:
Fixed in 09a829ca2bb72bd502177a4df54f06df5b640a51